### PR TITLE
fix docker shell typos

### PR DESCRIPTION
**Pull request by huanwei**
Signed-off-by: huanwei <huan@harmonycloud.cn>

## What is the purpose of the change

Fix tiny typos on shells in `rocketmq-externals/rocketmq-docker`. 

## Brief changelog

Changes includes:

```
modified:   rocketmq-docker/4.0.0-incubating/play.sh
modified:   rocketmq-docker/4.1.0-incubating/play.sh
modified:   rocketmq-docker/4.1.0-incubating/run.sh
```
## Verifying this change

Run the docker images to verify.

### CODE REVIEW

File: rocketmq-docker/4.0.0-incubating/play.sh [hunk 1]
```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
+
 sudo docker build -t apache/incubator-rocketmq-namesrv:4.0.0-incubating ./namesrv
 
 sudo docker build -t apache/incubator-rocketmq-broker:4.0.0-incubating ./broker
 
 sudo docker run -d -p 9876:9876 --name rmqnamesrv  apache/incubator-rocketmq-namesrv:4.0.0-incubating
 
-sudo docker run -d -p 10911:10911 -p 10909:10909 --name rmqbroker --link rmqnamesrv:rmqnamesrv -e "NAMESRV_ADDR=rmqnamesrv:9876" apache/incubator-rocketmq-broker:4.0.0-incubating
+sudo docker run -d -p 10911:10911 -p 10909:10909 --name rmqbroker --link rmqnamesrv:rmqnamesrv -e "NAMESRV_ADDR=rmqnamesrv:9876" apache/incubator-rocketmq-broker:4.0.0-incubating
```
Comment (vongosling) [inline rocketmq-docker/4.0.0-incubating/play.sh]: Could you help to update to the latest rocketmq ？

Comment (vongosling) [reply rocketmq-docker/4.0.0-incubating/play.sh]: see https://github.com/apache/rocketmq-externals/pull/75

Comment (huanwei) [reply rocketmq-docker/4.0.0-incubating/play.sh]: @vongosling Sure. I would be glad to do that.

Comment (huanwei) [reply rocketmq-docker/4.0.0-incubating/play.sh]: @vongosling For issue #75, I submitted PR #89.

Comment (vongosling) [reply rocketmq-docker/4.0.0-incubating/play.sh]: ok, I will close the pr, we could work on another pr together :-)

**Pull request closed without merging**